docs/backend/subscribers.md

```diff
@@ -11,8 +11,92 @@ myst:
 
 # Subscribers (event handlers)
 
-```{todo}
-Move content from:
--   https://5.docs.plone.org/external/plone.app.dexterity/docs/advanced/event-handlers.html
--   https://5.docs.plone.org/develop/addons/components/events.html
+In Plone 6, the event system provides a robust mechanism for developers to execute custom actions in response to various occurrences within the system.
+This capability is facilitated through the use of event subscribers, also known as event handlers, which listen for specific events and perform predefined operations when those events are triggered.
+
+## Overview of the Event System
+
+Plone's event system is built upon the Zope Component Architecture's `zope.event` package, enabling a publish-subscribe pattern where events are broadcasted, and subscribers (event handlers) respond accordingly.
+Key characteristics of this system include:
+
+- **Simplicity**: The event system is straightforward to implement and use.
+- **Non-deterministic Handler Execution Order**: The order in which event handlers are invoked is not specified and cannot be controlled.
+- **Non-cancelable Events**: Once an event is emitted, all associated handlers will receive it; events cannot be canceled.
+- **No Return Values**: Event handlers do not return values.
+- **Exception Handling**: Exceptions raised within an event handler can interrupt the request processing.
+
+Events are typically defined by interfaces and may carry pertinent information about the event.
```

Comment:
```suggestion
Events are matching interfaces and may carry relevant information about the event.
```
- defined by is not correct, they match an interface. This could be possibly expressed better.
- Actually I had to look up pertinent, keep the language simple.